src/tests/ftest/util/command_utils.py

```diff
@@ -714,7 +714,13 @@ def _get_json_result(self, sub_command_list=None, json_err=False,
             self.output_check = prev_output_check
             if json_err:
                 self.exit_status_exception = prev_exit_exception
-        return json.loads(self.result.stdout)
+        try:
+            return json.loads(self.result.stdout)
+        except json.decoder.JSONDecodeError as error:
+            if self.result.stdout:
+                raise error
+            # Error expected if the stdout is empty
+            return {}
```

Comment:
Couldn't we do this?
```suggestion
        if not self.result.stdout:
            return {}
        return json.loads(self.result.stdout)
```

Reply:
nit: I would swap the cases to avoid needing to negate in check

Reply:
Done.